src/Elastic.CommonSchema.Serilog/LogEventConverter.cs

```diff
@@ -91,7 +91,7 @@ private static Trace GetTrace(LogEvent logEvent) => !logEvent.TryGetScalarProper
 				? null
 				: new Transaction { Id = transactionId.Value.ToString() };
 
-		private static IDictionary<string, object> GetMetadata(LogEvent logEvent)
+		private static IDictionary<string, object> GetMetadata(LogEvent logEvent, ISet<string> propKeyFilter=null)
```

Comment:
```suggestion
		private static IDictionary<string, object> GetMetadata(LogEvent logEvent, ISet<string> logEventPropertiesToFilter)
```

Don't think this needs the `null` default value